test/distributed/_spmd/test_data_parallel.py

```diff
@@ -12,7 +12,7 @@
 from torch.distributed._tensor import Replicate
 from torch.nn.parallel import DistributedDataParallel as DDP
 from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
-from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.common_utils import run_tests  # noqa: TCH001
```

Comment:
cc @fegin This is because the spmd tests are all disabled per https://github.com/Skylion007/pytorch/commit/2f04aab1404383db1e1d160abf8149a1aeb8cccd